### PR TITLE
Update registrymirror doc

### PR DESCRIPTION
**Pull request by d8660091**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/getting-started/optional/registrymirror.md [hunk 1]
```diff
@@ -140,25 +139,3 @@ For [Mac](https://docs.docker.com/desktop/mac/#add-tls-certificates), you can fo
 {{% alert title="Note" color="primary" %}}
   You may need to restart Docker after adding the certificates.
 {{% /alert %}}
-
-## Registry configurations
```
Comment (jaxesn) [inline docs/content/en/docs/getting-started/optional/registrymirror.md:144]: shouldnt we keep this?  if the registry they are using requires these to exist ahead of time, don't they still need to do this?

File: docs/content/en/docs/getting-started/optional/registrymirror.md [hunk 2]
```diff
@@ -101,7 +100,6 @@ private registry.
 The `copy packages` must be used if you want to copy EKS Anywhere Curated Packages to your registry mirror.
 The `download images` command also pulls the cilium chart from `public.ecr.aws` and pushes it to the registry mirror. It requires the registry credentials for performing a login. Set the following environment variables for the login:
```
Comment (csplinter) [inline docs/content/en/docs/getting-started/optional/registrymirror.md:101]: Nit: capitalize "Cilium"

Comment (TerryHowe) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:101]:
```suggestion
The `download images` command also pulls the Cilium chart from `public.ecr.aws` and pushes it to the registry mirror. It requires the registry credentials for performing a login. Set the following environment variables for the login:
```

File: docs/content/en/docs/getting-started/optional/registrymirror.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ spec:
 * __Example__: ```port: 443```
 
 ### __ociNamespaces__ (optional)
-* __Description__: mapping from upstream registries to the locations of the private registry. When specified, the artifacts pulled from an upstream registry will be put in its corresponding location/namespace in the private registry. The target location/namespace must be already existing.
+* __Description__: when you need to mirror multiple registries, you can map the upstream registries to "namespaces" of the mirror.
```
Comment (chrisnegus) [inline docs/content/en/docs/getting-started/optional/registrymirror.md:53]: Is this a one-to-one mapping of a registry to a namespace? If so, I'd suggest wording like:
```suggestion
* __Description__: when you need to mirror multiple registries, you can map each upstream registry to the "namespace" of its mirror.
```
Let me know if I misunderstand you.

**Review by chrisnegus — COMMENTED** [review]
I just have one question about the wording of registryMirrorConfiguration.

File: docs/content/en/docs/getting-started/optional/registrymirror.md (hunk 1 above)
Comment (abhinavmpandey08) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:144]: yeah I think this section is still applicable

Comment (d8660091) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:144]: Is it only required for harbor? I never created those projects for my docker registry server running locally?

File: docs/content/en/docs/getting-started/optional/registrymirror.md (hunk 3 above)
Comment (d8660091) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:53]: sounds better to me, thanks!

File: docs/content/en/docs/getting-started/optional/registrymirror.md (hunk 1 above)
Comment (abhinavmpandey08) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:144]: i know it's needed for harbor and i think another one as well, maybe artifactory

Comment (jaxesn) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:144]: yeah we should keep this.

Comment (d8660091) [reply docs/content/en/docs/getting-started/optional/registrymirror.md]: added back

File: docs/content/en/docs/getting-started/optional/registrymirror.md [hunk 4]
```diff
@@ -152,8 +151,7 @@ isovalent
 cilium-chart
 ```
 
-For example, if a registry is available at `private-registry.local`, then the following 
-projects will have to be created:
+For example, if a registry is available at `private-registry.local`, then the following projects will have to be created:
```
Comment (drewvanstone) [inline docs/content/en/docs/getting-started/optional/registrymirror.md:154]: There is nothing following this statement? Is there supposed to be an example here that's missing?

**Review by drewvanstone — CHANGES_REQUESTED** [review]
Looks like we're missing an example or something. Everything else looks good from a writing perspective. Haven't tested it for correctness.

File: docs/content/en/docs/getting-started/optional/registrymirror.md (hunk 4 above)
Comment (d8660091) [reply docs/content/en/docs/getting-started/optional/registrymirror.md:154]: is it wrapped by github UI?

**Review by drewvanstone — APPROVED** [review]
Yep you're right. Github preview was truncating the doc

**Pull request merged**